scss/_tag.scss

```diff
@@ -0,0 +1,129 @@
+  --#{$prefix}tag-active-color: #{$tag-active-color};
+  --#{$prefix}tag-active-decoration-color: #{$tag-active-decoration-color};
+  --#{$prefix}tag-disabled-color: #{$tag-disabled-color};
+  --#{$prefix}tag-outline-color: var(--#{$prefix}link-hover-color);
+
+  display: inline-flex;
+  gap: var(--#{$prefix}tag-gap);
+  align-items: center;
+  padding: subtract(var(--#{$prefix}tag-padding-y), calc(var(--#{$prefix}tag-font-shift) * .5)) var(--#{$prefix}tag-padding-x) add(var(--#{$prefix}tag-padding-y), calc(var(--#{$prefix}tag-font-shift) * .5)); // stylelint-disable-line function-disallowed-list
+  @include font-size(var(--#{$prefix}tag-font-size));
+  font-weight: var(--#{$prefix}tag-font-weight);
+  line-height: add(var(--#{$prefix}tag-font-size), .125rem);
+  color: inherit;
+  letter-spacing: var(--#{$prefix}tag-font-spacing);
+  background-color: transparent;
+  border: var(--#{$prefix}tag-border-width) solid var(--#{$prefix}tag-border-color);
+  outline-offset: $spacer;
+  @include transition($transition-focus);
+  @include border-radius(var(--#{$prefix}tag-border-radius), var(--#{$prefix}tag-border-radius));
+
+  img,
+  svg {
+    width: var(--#{$prefix}tag-icon-size);
+    min-width: var(--#{$prefix}tag-icon-size); // Here to avoid weird behavior on wrap
+    height: var(--#{$prefix}tag-icon-size);
+    margin: add(calc(var(--#{$prefix}tag-icon-size) * -.5), var(--#{$prefix}tag-font-shift)) 0 calc(var(--#{$prefix}tag-icon-size) * -.5) var(--#{$prefix}tag-icon-margin-start); // stylelint-disable-line function-disallowed-list
+  }
+
+  .close {
+    position: relative;
+    width: var(--#{$prefix}tag-close-size);
+    min-width: var(--#{$prefix}tag-close-size); // Here to avoid weird behavior on wrap
+    height: var(--#{$prefix}tag-close-size);
+    margin: add(calc(var(--#{$prefix}tag-close-size) * -.5), var(--#{$prefix}tag-font-shift)) var(--#{$prefix}tag-close-margin-end) calc(var(--#{$prefix}tag-close-size) * -.5) var(--#{$prefix}tag-close-margin-start); // stylelint-disable-line function-disallowed-list
+    color: inherit;
+    background-color: transparent;
+    border: 0;
+    outline-offset: $spacer;
+    @include transition($transition-focus);
+    @include border-radius(var(--#{$prefix}tag-border-radius), var(--#{$prefix}tag-border-radius));
+
+    &::after {
+      position: absolute;
+      width: 100%;
+      height: 100%;
+      content: "";
+      background-color: currentcolor;
+      transform: translate(-50%, -50%);
+      mask-image: escape-svg(url("data:image/svg+xml,<svg xmlns='http://www.w3.org/2000/svg' viewBox='0 0 30 30'><path d='M24.536 20.298a1.499 1.499 0 0 1 0 2.12l-2.119 2.118a1.499 1.499 0 0 1-2.119 0L15 19.238l-5.298 5.298a1.499 1.499 0 0 1-2.12 0l-2.118-2.119a1.499 1.499 0 0 1 0-2.119L10.762 15 5.464 9.702a1.499 1.499 0 0 1 0-2.12l2.119-2.118a1.499 1.499 0 0 1 2.119 0L15 10.762l5.298-5.298a1.499 1.499 0 0 1 2.12 0l2.118 2.119a1.499 1.499 0 0 1 0 2.119L19.238 15z'/></svg>"));
```

Comment:
Why not using the close icon already defined in _variables.scss?

Reply:
The 2 svgs are slightly different, so I added this one, but you're right, it should be located at the same place.